src/cli/index.ts

```diff
@@ -248,5 +257,12 @@ installation.
         `Supported styles: "name", "path".`);
   }
 
+  const packageType = options['package-type'];
+  if (packageType !== 'element' && packageType !== 'application') {
```

Comment:
If you had a `packageTypes` variable you could use it all over the place and do `!packageTypes.includes(packageType)` and like `Supported types: "${packageTypes.join('", "')}"` and stuff but this is fine.